openshift/topics/advanced_concepts.adoc

```diff
@@ -23,22 +28,21 @@ $ oc policy add-role-to-user view system:serviceaccount:$(oc project -q):default
 [[Configuring-Keystores]]
 ==== Creating HTTPS and JGroups Keystores, and Truststore for the {project_name} Server
 
-The {project_name} application templates using xref:../introduction/introduction.adoc#passthrough-templates[passthrough TLS termination] require:
+In this procedure, the *_openssl_* toolkit is used to generate a CA certificate to sign the HTTPS keystore, and create a truststore for the {project_name} server. *_keytool_*, a package *included with the Java Development Kit*, is then used to generate self-signed certificates for these keystores.
 
-* An xref:create-https-keystore[HTTPS keystore] used for encryption of https traffic,
-* The xref:create-jgroups-keystore[JGroups keystore] used for encryption of JGroups communications between nodes in the cluster, and
-* xref:create-server-truststore[{project_name} server truststore] used for securing the {project_name} requests
+The {project_name} application templates using xref:../introduction/introduction.adoc#reencrypt-templates[re-encryption TLS termination] do not *require* or *expect* the HTTPS and JGroups keystores and {project_name} server truststore to be prepared beforehand. The templates use OpenShift's internal link:{ocpdocs_serving_x509_secrets_link}[service serving x509 certificate secrets] to automatically create the HTTPS and JGroups keystores.
```

Comment:
Maybe here is the right place to add a note / IOW change:
`The {project_name} ... to be prepared beforehand. The templates...` 

to

```
The {project_name} ... to be prepared beforehand. 

Note:
If you want to provision the {project_name} server using existing HTTPS / JGroups keystores, use some of the available passthrough templates instead.

The re-encrypt templates use the OpenShift's internal ....
```